core-web/libs/dotcms-scss/angular/dotcms-theme/components/_galleria.scss

```diff
@@ -0,0 +1 @@
+//TODO: Implement Galleria styles.
```

Comment:
Remove?

Reply:
Medhi proposed creating a new ticket for each "TODO" instead of leaving a comment in the code.